test/features/edgeconnect/edgeconnect.go

```diff
@@ -14,6 +13,7 @@ import (
 	controller "github.com/Dynatrace/dynatrace-operator/pkg/controllers/edgeconnect"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/edgeconnect"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/rand"
```

Comment:
this was used for the network-zone tests(and is actually dead code 😅 ), that were still flaky, is it the best idea to use it ? 😅 

what is the problem with the `uuid` package ?

Reply:
ooops, one second.

Reply:
but we don't use uuid anywhere...

Reply:
so we implement/maintain an inferior inhouse thing for the same purpose while it's already part of our go.mod 😅

Reply:
let's use the uuid for a proper subdomain, instead of a single int with high chance of collision

Reply:
fixed in e82d1097

also removed dead code.